### PR TITLE
*: support Super_priv

### PR DESCRIPTION
**Pull request by tiancaiamao**
Add `Super_priv` to `mysql.user` table and use it.

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

**Comment by tiancaiamao** [comment]
PTAL @shenli @zimulala

File: bootstrap.go [hunk 1]
```diff
@@ -47,6 +47,7 @@ const (
 		Grant_priv		ENUM('N','Y') NOT NULL  DEFAULT 'N',
 		Alter_priv		ENUM('N','Y') NOT NULL  DEFAULT 'N',
 		Show_db_priv		ENUM('N','Y') NOT NULL  DEFAULT 'N',
+		Super_priv		ENUM('N','Y') NOT NULL  DEFAULT 'N',
 		Execute_priv		ENUM('N','Y') NOT NULL  DEFAULT 'N',
 		Index_priv		ENUM('N','Y') NOT NULL  DEFAULT 'N',
 		Create_user_priv	ENUM('N','Y') NOT NULL  DEFAULT 'N',
```
Comment (zimulala) [inline bootstrap.go]: Unify the format.

File: privilege/privileges/cache_test.go [hunk 2]
```diff
@@ -53,11 +53,11 @@ func (s *testCacheSuite) TestLoadUserTable(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(len(p.User), Equals, 0)
 
-	// Host | User | Password | Select_priv | Insert_priv | Update_priv | Delete_priv | Create_priv | Drop_priv | Grant_priv | Alter_priv | Show_db_priv | Execute_priv | Index_priv | Create_user_priv
-	mustExec(c, se, `INSERT INTO mysql.user VALUES ("%", "root", "", "Y", "N", "N", "N", "N", "N", "N", "N", "N", "N", "N", "N")`)
-	mustExec(c, se, `INSERT INTO mysql.user VALUES ("%", "root1", "admin", "N", "Y", "N", "N", "N", "N", "N", "N", "N", "N", "N", "N")`)
-	mustExec(c, se, `INSERT INTO mysql.user VALUES ("%", "root11", "", "N", "N", "Y", "N", "N", "N", "N", "N", "Y", "N", "N", "N")`)
-	mustExec(c, se, `INSERT INTO mysql.user VALUES ("%", "root111", "", "N", "N", "N", "N", "N", "N", "N", "N", "Y", "Y", "Y", "Y")`)
+	// Host | User | Password | Select_priv | Insert_priv | Update_priv | Delete_priv | Create_priv | Drop_priv | Grant_priv | Alter_priv | Show_db_priv | Super_priv | Execute_priv | Index_priv | Create_user_priv
```
Comment (zimulala) [inline privilege/privileges/cache_test.go]: Your operation of `add column` doesn't use `after`, so this schema will be `Host | User | Password | Select_priv | Insert_priv | Update_priv | Delete_priv | Create_priv | Drop_priv | Grant_priv | Alter_priv | Show_db_priv | Execute_priv | Index_priv | Create_user_priv | Super_priv`

File: bootstrap.go (hunk 1 above)
Comment (tiancaiamao) [inline bootstrap.go]: Seems it github's bug, it's actually formated.

**Comment by zimulala** [comment]
LGTM

**Pull request merged**